spec/fixtures/scripts/success.ps1

```diff
@@ -0,0 +1,8 @@
+foreach ($i in $args)
+{
+    Write-Output "arg: $i"
+}
+
+Write-Output "standard output"
+
+# Write-Error "standard error"
```

Comment:
@Iristyle need to talk to you about this

Reply:
Out of curiosity, what happens if you `throw "error!"` instead?